target/linux/pistachio/base-files/etc/booard.d/02_network

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+
+. /lib/functions/uci-defaults.sh
+. /lib/pistachio.sh
+
+board_config_update
+
+board=$(pistachio_board_name)
+
+case "$board" in
+"Marduk")
+	lan_mac=$(mtd_get_mac_ascii uEnv ethaddr)
```

Comment:
We don't need this line anymore right? Because mac will already be set in the device tree.